content/docs/user-guide/structure-of-remote-cache-by-example.md

```diff
@@ -0,0 +1,251 @@
+you could further check the contents of the remote cache.
+Here's what the S3 cache looked like on my system:
+
+```dvc
+$ aws s3 ls s3://path/to/your/cache/
+                           PRE 61/
+2022-02-20 13:16:49          0
+```
+
+One folder `61/` has been created, and if you look inside this, the folder contains only a single file with an obfuscated (hexademical) name.
+
+So what's going on?
+
+The answer, of course, is that the four data files were identical and so DVC concluded that only one copy of the files really needed to be uploaded.
+The name `61/` is an internal name for the file used to implement an indirection.
+So the file `data1` on my local computer references `61/` on the remote cache, the file `data2` on my local computer also references `61/`,
+and so on.
+
+DVC determines which files are identical by comparing their MD5 hashes, and you can confirm this by looking at the `.dvc` files for the four data files yourself: 
+
+```dvc
+$ cat *.dvc
+outs:
+- md5: 6137cde4893c59f76f005a8123d8e8e6
+  size: 5
+  path: data1
+outs:
+- md5: 6137cde4893c59f76f005a8123d8e8e6
+  size: 5
+  path: data2
+outs:
+- md5: 6137cde4893c59f76f005a8123d8e8e6
+  size: 5
+  path: data3
+outs:
+- md5: 6137cde4893c59f76f005a8123d8e8e6
+  size: 5
+  path: data4
+```
+
+Graphically, the project folder and the remote cache have this structure:
+
+![Data in project and remote cache](../../../static/img/project_remote_1.png)
+
+## Adding new data and how the remote cache changes
+
+Let's add two more data files to our repository, but this time make them  different to the first four data files. We should also make sure DVC tracks the new files and add the corresponding `.dvc` files with git: 
+
+```dvc
+$ echo data3636363 > data5
+$ echo blahblahblah > data6
+$ dvc add data5
+$ dvc add data6
+$ git add data5.dvc
+$ git add data6.dvc
+$ git add .gitignore
+$ git commit -m "added more data"
+```
+
+So... at this point we will have four identical data files (`data1`...`data4`) and two new data files, `data5` and `data6`.
+Based on what happened the last time we pushed data to the remote cache,
+we can predict that since the two new data files are different to the others and two each other,
+they will both need to be pushed to the remote cache.
+This, in fact, is exactly what happens:
+
+```dvc
+$ dvc push -r s3cache
+2 files pushed
+```
+
+The remote cache will now have three different items, which you can see:
+
+```dvc
+$ aws s3 ls s3://path/to/your/cache/
+                           PRE 61/
+                           PRE 8c/
+                           PRE e1/
+2022-02-20 13:16:49          0
+```
+
+Graphically speaking, the setup will now look like this:
+
+![Data in project and remote cache after adding more data](../../../static/img/project_remote_2.png)
+
+
+## Modifying existing data 
+
+Next, let's change one of the data files that used to be a duplicate and push the changes to the remote cache:
```

Comment:
Minor: I would've personally combined this with the previous section since you can add and modify in the same step. But that's me (I tend to summarize as much as possible).